src/evaluation/manipulability_moveit.cpp

```diff
@@ -179,6 +180,34 @@ reach::Evaluator::ConstPtr ManipulabilityRatioFactory::create(const YAML::Node&
   return std::make_shared<ManipulabilityRatio>(model, planning_group, jacobian_row_subset);
 }
 
+double ManipulabilityRatioSigmoid::calculateScore(const Eigen::MatrixXd& jacobian_singular_values) const
+{
+  double manipulability = jacobian_singular_values.array().prod();
+  double manipulability_ratio = jacobian_singular_values.minCoeff() / jacobian_singular_values.maxCoeff();
+  double manipulability_scaled = std::sqrt(manipulability);
+  double manipulability_measure = std::sqrt(manipulability_ratio);
```

Comment:
Remove unused variables

```suggestion
  double manipulability_ratio = jacobian_singular_values.minCoeff() / jacobian_singular_values.maxCoeff();
  double manipulability_measure = std::sqrt(manipulability_ratio);
```